app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return phrase.lower().count(letter.lower())
```

Comment:
The function `count_occurrences` is expected to return an integer, representing the count of occurrences of `letter` in `phrase`. However, the current implementation returns a float because the `count` method is used correctly, but there is no need for division. Simply return the result of `phrase.lower().count(letter.lower())` to get an integer.